src/style.css

```diff
@@ -88,11 +111,18 @@ h3 {
+  background-attachment: fixed;
+  background-size: cover;
+}
+
 .main-container {
-  /* display: inline; */
   margin: 0 auto;
   max-width: 700px;
-  border: 1px grey solid;
+
+  /* border: 1px grey solid; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.